modules/react/checkbox/lib/Checkbox.tsx

```diff
@@ -13,7 +13,7 @@ import canvas, {
   borderRadius,
   colors,
   inputColors,
-  spacingNumbers as spacing,
```

Comment:
This is a very weird thing we were doing. I've only found this in CK (and old CK versions in GHE). There's no reason for the alias, but it threw an error and required a manual update.